extensions/adobe/experience/adcloud/searchadvertisingplatform.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertisingplatform",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Platform",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Information about Search Advertising Platform.",
+  "definitions": {
+    "searchadvertisingplatform": {
+      "properties": {
+        "https://ns.adobe.com/experience/adcloud/adPlatform": {
```

Comment:
Why did we use "xdm" namespaced fields in the classes above, but adcloud name spaced fields here? Not saying one or the other is incorrect, just wanting to understand the reasoning.

Reply:
Initial thought was this class will be specific to ad-cloud as it is not a generic entity in search advertising market.  But it can leave in xdm namespace too.  I have modified it to be in xdm namespace